articlequality/extractors/ptwiki.py

```diff
@@ -10,12 +10,12 @@ def from_template(template):
     project_name = normalize_project_name(template.name)
     if project_name == "marca de projeto":
         labels = extract_labels(template)
-        if len(labels) >= 1:
-            for project, label in labels[1:]:
-                yield (normalize_project_name(project), label)
+        if len(labels) > 0:
+            project, label = labels[0]
+            yield (normalize_project_name(project), label)
 
 
-PROJECT_LABEL = re.compile(r"([^\|\{\{\}\}]+)\|([0-5\*])", re.I)
+PROJECT_LABEL = re.compile(r"([^\|\{\{\}\}]+)\|([0-6])", re.I)
```

Comment:
This will not match articles whose quality is set to "AD" or "AB", which are aliases for "6" and "5" respectively. See https://regex101.com/r/XdXRkk/1

Reply:
BTW: wouldn't be safer to use `mwparserfromhell`'s methods to get the parameters and their values, instead of creating our own regexes? For enwiki, it was used e.g.:
https://github.com/wikimedia/articlequality/blob/c23aacd586b0889ea0ac26d32a9bb6b223821e88/articlequality/extractors/enwiki.py#L14
and for frwiki, e.g.:
https://github.com/wikimedia/articlequality/blob/c23aacd586b0889ea0ac26d32a9bb6b223821e88/articlequality/extractors/frwiki.py#L16

For ptwiki, the quality is provided by users as the first unnamed parameter (although the template also accepts "qualidade" as an alias to this first parameter), so we can use `.get(1).value` as in
```lang=python
import mwparserfromhell
wikicode = mwparserfromhell.parse('{{Marca de projeto|AD|Wrestling|3}}')
templates = wikicode.filter_templates()
template = templates[0]
print(template.params)
# ['AD', 'Wrestling', '3']
print(template.get(1).value)
# AD
```
to obtain the user's acessment of the quality of the article.